server/features/search_planning.feature

```diff
@@ -260,10 +260,15 @@ Feature: Planning Search
         ]}
         """
         When we get "/events_planning_search?repo=planning&only_future=false&language=en"
-        Then we get list with 1 items
+        Then we get list with 6 items
```

Comment:
Same with this one, changing the Planning item's language to a different value than the system default and changing this URL to reflect.